### PR TITLE
Magic number cleanup part 13

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
A new set of files that were not part of the first 12 initial PRs about the magic numbers.
But when applying the PR feedback, these files had the same issues as the magic number PRs 1 through 12.
So the changes also need to be merged into develop.

### CODE REVIEW

File: packages/app/src/style/preset.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export const preset = {
 
 export const nestedHtml = {
   ...spacingStyle(3),
-  /** p: inherit p-styles from the container */
+  /** padding: inherit p-styles from the container */
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/style/preset.ts:99]: You might want te revert this change, again. 😜

**Review by VWSCoronaDashboard26 — APPROVED** [review]
Pre-approved, but have a look at my single non-blocking comment.

File: packages/app/src/style/preset.ts (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/style/preset.ts]: Reverted. Too many PRs will solve some issues but creates other issues.

**Review by VWSCoronaDashboard27 — APPROVED** [review]
